zib/StructureDefinitions/zib-LaboratoryTestResult.Specimen-BodyStructure.xml

```diff
@@ -0,0 +1,56 @@
+  <id value="zib-LaboratoryTestResult.Specimen-BodyStructure" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-LaboratoryTestResult.Specimen-BodyStructure" />
+  <name value="ZibLaboratoryTestResultSpecimenBodyStructure" />
+  <title value="zib LaboratoryTestResult.Specimen BodyStructure" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The body structure where the specimen material is obtained." />
+  <purpose value="This BodyStructure resource represents the AnatomicalLocation and Morphology concepts of the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) LaboratoryTestResult v7.0 (2024)](https://zibs.nl/wiki/LaboratoryTestResult-v7.0(2024EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-laboratorytestresult-v7.0-2024EN" />
+    <uri value="https://zibs.nl/wiki/LaboratoryTestResult-v7.0(2024EN)" />
+    <name value="zib LaboratoryTestResult-v7.0(2024EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-anatomicallocation-v1.0.3-2024EN" />
+    <uri value="https://zibs.nl/wiki/AnatomicalLocation-v1.0.3(2024EN)" />
+    <name value="zib AnatomicalLocation-v1.0.3(2024EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="BodyStructure" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-AnatomicalLocation-BodyStructure" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="BodyStructure">
+      <path value="BodyStructure" />
+      <definition value="The zib AnatomicalLocation is mapped to both a profile on CodeableConcept (&lt;http://nictiz.nl/fhir/StructureDefinition/zib-AnatomicalLocation&gt;) and this BodyStructure profile. While the CodeableConcept profile covers anatomical location, the BodyStructure resource is used to capture both the location and its laterality.&#xA;&#xA;When laterality and/or morphology are required, this BodyStructure profile should be used. In cases where laterality and morphology are not needed, the CodeableConcept can be used within the resource.&#xD;&#xA;&#xD;&#xA;**Note:** Receiving systems must support both the BodyStructure resource and the CodeableConcept in host resources, as either mechanism may be employed. Providing systems can select the appropriate mechanism based on their data requirements." />
```

Comment:
This sentence is added I believe:

> When laterality and/or morphology are required, this BodyStructure profile should be used. In cases where laterality and morphology are not needed, the CodeableConcept can be used within the resource.

This profile is pretty specific to LaboratoryTestResult. So we can add more specific information. For example instead of "the CodeableConcept can be used within the resource." it could be something as the CodeableConcept `Specimen.name` (profife name) can be used.